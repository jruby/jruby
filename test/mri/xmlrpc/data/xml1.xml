<?xml version="1.0" encoding="ISO-8859-1"?><methodResponse><params><param><value><array><data><value><struct><member><name>objectName</name><value>AdministratorDO</value></member><member><name>adminId</name><value>hbaker</value></member><member><name>email</name><value>hbaker@yahoo.com</value></member><member><name>telephone2</name><value>1-800-445-2588</value></member><member><name>telephone1</name><value>1-508-791-1267</value></member><member><name>OID</name><value>1</value></member><member><name>password</name><value>p1111</value></member><member><name>lastName</name><value>Baker</value></member><member><name>subscriber</name><value>MegaCorp</value></member></struct></value><value><struct><member><name>objectName</name><value>AdministratorDO</value></member><member><name>adminId</name><value>adragon</value></member><member><name>email</name><value>adragon@yahoo.com</value></member><member><name>telephone2</name><value>1-800-445-2588</value></member><member><name>telephone1</name><value>1-781-789-9089</value></member><member><name>OID</name><value>3</value></member><member><name>password</name><value>p3333</value></member><member><name>lastName</name><value>Dragon</value></member><member><name>subscriber</name><value>CornerStore</value></member></struct></value><value><struct><member><name>objectName</name><value>AdministratorDO</value></member><member><name>adminId</name><value>mrodman</value></member><member><name>email</name><value>mrodman@yahoo.com</value></member><member><name>telephone2</name><value>1-800-445-2588</value></member><member><name>telephone1</name><value>1-617-789-1890</value></member><member><name>OID</name><value>4</value></member><member><name>password</name><value>p4444</value></member><member><name>lastName</name><value>Rodman</value></member><member><name>subscriber</name><value>Cyberdyne</value></member></struct></value><value><struct><member><name>objectName</name><value>AdministratorDO</value></member><member><name>adminId</name><value>mjordan</value></member><member><name>email</name><value>mjordan@yahoo.com</value></member><member><name>telephone2</name><value>1-800-445-2588</value></member><member><name>telephone1</name><value>1-617-890-7897</value></member><member><name>OID</name><value>5</value></member><member><name>password</name><value>p5555</value></member><member><name>lastName</name><value>Jordan</value></member><member><name>subscriber</name><value>StarSports</value></member></struct></value><value><struct><member><name>objectName</name><value>AdministratorDO</value></member><member><name>adminId</name><value>gpippen</value></member><member><name>email</name><value>gpippen@yahoo.com</value></member><member><name>telephone2</name><value>1-800-445-2588</value></member><member><name>telephone1</name><value>1-781-789-9876</value></member><member><name>OID</name><value>6</value></member><member><name>password</name><value>p6666</value></member><member><name>lastName</name><value>Pippen</value></member><member><name>subscriber</name><value>GreatBooks</value></member></struct></value><value><struct><member><name>objectName</name><value>AdministratorDO</value></member><member><name>adminId</name><value>aandrew</value></member><member><name>email</name><value>aandrew@yahoo.com</value></member><member><name>telephone2</name><value>1-800-445-2588</value></member><member><name>telephone1</name><value>1-781-678-8970</value></member><member><name>OID</name><value>7</value></member><member><name>password</name><value>p7777</value></member><member><name>lastName</name><value>Andhrew</value></member><member><name>subscriber</name><value>AxisChemicals</value></member></struct></value><value><struct><member><name>objectName</name><value>AdministratorDO</value></member><member><name>adminId</name><value>tvincent</value></member><member><name>email</name><value>tvincent@yahoo.com</value></member><member><name>telephone2</name><value>1-800-445-2588</value></member><member><name>telephone1</name><value>1-786-897-8908</value></member><member><name>OID</name><value>8</value></member><member><name>password</name><value>p8888</value></member><member><name>lastName</name><value>Vincent</value></member><member><name>subscriber</name><value>MediaShop</value></member></struct></value><value><struct><member><name>objectName</name><value>AdministratorDO</value></member><member><name>adminId</name><value>krichard</value></member><member><name>email</name><value>krichard@yahoo.com</value></member><member><name>telephone2</name><value>1-800-445-2588</value></member><member><name>telephone1</name><value>1-508-789-6789</value></member><member><name>OID</name><value>9</value></member><member><name>password</name><value>p9999</value></member><member><name>lastName</name><value>Richard</value></member><member><name>subscriber</name><value>SmartShop</value></member></struct></value><value><struct><member><name>objectName</name><value>AdministratorDO</value></member><member><name>adminId</name><value>gcornell</value></member><member><name>email</name><value>gconell@yahoo.com</value></member><member><name>telephone2</name><value>1-800-445-2588</value></member><member><name>telephone1</name><value>1-617-789-8979</value></member><member><name>OID</name><value>10</value></member><member><name>password</name><value>paaaa</value></member><member><name>lastName</name><value>Cornell</value></member><member><name>subscriber</name><value>HomeNeeds</value></member></struct></value><value><struct><member><name>objectName</name><value>AdministratorDO</value></member><member><name>adminId</name><value>shorstmann</value></member><member><name>email</name><value>shorstmann@yahoo.com</value></member><member><name>telephone2</name><value>1-800-445-2588</value></member><member><name>telephone1</name><value>1-508-791-1267</value></member><member><name>OID</name><value>11</value></member><member><name>password</name><value>p1111</value></member><member><name>lastName</name><value>HorstMann</value></member><member><name>subscriber</name><value>MegaCorp</value></member></struct></value><value><struct><member><name>objectName</name><value>AdministratorDO</value></member><member><name>adminId</name><value>rbob</value></member><member><name>email</name><value>rbob@yahoo.com</value></member><member><name>telephone2</name><value>1-800-445-2588</value></member><member><name>telephone1</name><value>1-781-789-9089</value></member><member><name>OID</name><value>13</value></member><member><name>password</name><value>p3333</value></member><member><name>lastName</name><value>Bob</value></member><member><name>subscriber</name><value>CornerStore</value></member></struct></value><value><struct><member><name>objectName</name><value>AdministratorDO</value></member><member><name>adminId</name><value>speter</value></member><member><name>email</name><value>speter@yahoo.com</value></member><member><name>telephone2</name><value>1-800-445-2588</value></member><member><name>telephone1</name><value>1-617-789-1890</value></member><member><name>OID</name><value>14</value></member><member><name>password</name><value>p4444</value></member><member><name>lastName</name><value>Peter</value></member><member><name>subscriber</name><value>Cyberdyne</value></member></struct></value><value><struct><member><name>objectName</name><value>AdministratorDO</value></member><member><name>adminId</name><value>pnovak</value></member><member><name>email</name><value>pnovak@yahoo.com</value></member><member><name>telephone2</name><value>1-800-445-2588</value></member><member><name>telephone1</name><value>1-617-890-7897</value></member><member><name>OID</name><value>15</value></member><member><name>password</name><value>p5555</value></member><member><name>lastName</name><value>Novak</value></member><member><name>subscriber</name><value>StarSports</value></member></struct></value><value><struct><member><name>objectName</name><value>AdministratorDO</value></member><member><name>adminId</name><value>pnancy</value></member><member><name>email</name><value>pnancy@yahoo.com</value></member><member><name>telephone2</name><value>1-800-445-2588</value></member><member><name>telephone1</name><value>1-781-789-9876</value></member><member><name>OID</name><value>16</value></member><member><name>password</name><value>p6666</value></member><member><name>lastName</name><value>Nancy</value></member><member><name>subscriber</name><value>GreatBooks</value></member></struct></value><value><struct><member><name>objectName</name><value>AdministratorDO</value></member><member><name>adminId</name><value>hmichel</value></member><member><name>email</name><value>hmichel@yahoo.com</value></member><member><name>telephone2</name><value>1-800-445-2588</value></member><member><name>telephone1</name><value>1-781-678-8970</value></member><member><name>OID</name><value>17</value></member><member><name>password</name><value>p7777</value></member><member><name>lastName</name><value>Michel</value></member><member><name>subscriber</name><value>AxisChemicals</value></member></struct></value><value><struct><member><name>objectName</name><value>AdministratorDO</value></member><member><name>adminId</name><value>kdavid</value></member><member><name>email</name><value>kdavid@yahoo.com</value></member><member><name>telephone2</name><value>1-800-445-2588</value></member><member><name>telephone1</name><value>1-786-897-8908</value></member><member><name>OID</name><value>18</value></member><member><name>password</name><value>p8888</value></member><member><name>lastName</name><value>David</value></member><member><name>subscriber</name><value>MediaShop</value></member></struct></value><value><struct><member><name>objectName</name><value>AdministratorDO</value></member><member><name>adminId</name><value>pvalnoor</value></member><member><name>email</name><value>pvalnoor@yahoo.com</value></member><member><name>telephone2</name><value>1-800-445-2588</value></member><member><name>telephone1</name><value>1-508-789-6789</value></member><member><name>OID</name><value>19</value></member><member><name>password</name><value>p9999</value></member><member><name>lastName</name><value>Valnoor</value></member><member><name>subscriber</name><value>SmartShop</value></member></struct></value><value><struct><member><name>objectName</name><value>AdministratorDO</value></member><member><name>adminId</name><value>wsmith</value></member><member><name>email</name><value>wsmith@yahoo.com</value></member><member><name>telephone2</name><value>1-800-445-2588</value></member><member><name>telephone1</name><value>1-617-789-8979</value></member><member><name>OID</name><value>20</value></member><member><name>password</name><value>paaaa</value></member><member><name>lastName</name><value>Smith</value></member><member><name>subscriber</name><value>HomeNeeds</value></member></struct></value><value><struct><member><name>objectName</name><value>AdministratorDO</value></member><member><name>adminId</name><value>gcaral</value></member><member><name>email</name><value>gcaral@yahoo.com</value></member><member><name>telephone2</name><value>1-800-445-2588</value></member><member><name>telephone1</name><value>1-781-789-9876</value></member><member><name>OID</name><value>21</value></member><member><name>password</name><value>p6666</value></member><member><name>lastName</name><value>Caral</value></member><member><name>subscriber</name><value>MegaCorp</value></member></struct></value><value><struct><member><name>objectName</name><value>AdministratorDO</value></member><member><name>adminId</name><value>phillary</value></member><member><name>email</name><value>phillary@yahoo.com</value></member><member><name>telephone2</name><value>1-800-445-2588</value></member><member><name>telephone1</name><value>1-786-897-8908</value></member><member><name>OID</name><value>23</value></member><member><name>password</name><value>p8888</value></member><member><name>lastName</name><value>Hillary</value></member><member><name>subscriber</name><value>CornerStore</value></member></struct></value><value><struct><member><name>objectName</name><value>AdministratorDO</value></member><member><name>adminId</name><value>bphilip</value></member><member><name>email</name><value>bphilip@yahoo.com</value></member><member><name>telephone2</name><value>1-800-445-2588</value></member><member><name>telephone1</name><value>1-508-789-6789</value></member><member><name>OID</name><value>24</value></member><member><name>password</name><value>p9999</value></member><member><name>lastName</name><value>Philip</value></member><member><name>subscriber</name><value>Cyberdyne</value></member></struct></value><value><struct><member><name>objectName</name><value>AdministratorDO</value></member><member><name>adminId</name><value>sandrea</value></member><member><name>email</name><value>sandrea@yahoo.com</value></member><member><name>telephone2</name><value>1-800-445-2588</value></member><member><name>telephone1</name><value>1-617-789-8979</value></member><member><name>OID</name><value>25</value></member><member><name>password</name><value>paaaa</value></member><member><name>lastName</name><value>Andrea</value></member><member><name>subscriber</name><value>StarSports</value></member></struct></value><value><struct><member><name>objectName</name><value>AdministratorDO</value></member><member><name>adminId</name><value>s4</value></member><member><name>email</name><value>null</value></member><member><name>telephone2</name><value>null</value></member><member><name>telephone1</name><value>null</value></member><member><name>OID</name><value>26</value></member><member><name>password</name><value>s4</value></member><member><name>lastName</name><value>null</value></member><member><name>subscriber</name><value>s4</value></member></struct></value><value><struct><member><name>objectName</name><value>AdministratorDO</value></member><member><name>adminId</name><value>admin</value></member><member><name>email</name><value></value></member><member><name>telephone2</name><value></value></member><member><name>telephone1</name><value></value></member><member><name>OID</name><value>82</value></member><member><name>password</name><value>admin</value></member><member><name>lastName</name><value>administrator</value></member><member><name>subscriber</name><value>BigBank</value></member></struct></value></data></array></value></param></params></methodResponse>
