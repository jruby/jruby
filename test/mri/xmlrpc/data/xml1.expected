--- 
- true
- 
  - 
    subscriber: MegaCorp
    lastName: Baker
    telephone1: 1-508-791-1267
    telephone2: 1-800-445-2588
    password: p1111
    OID: "1"
    email: hbaker@yahoo.com
    adminId: hbaker
    objectName: AdministratorDO
  - 
    subscriber: CornerStore
    lastName: Dragon
    telephone1: 1-781-789-9089
    telephone2: 1-800-445-2588
    password: p3333
    OID: "3"
    email: adragon@yahoo.com
    adminId: adragon
    objectName: AdministratorDO
  - 
    subscriber: Cyberdyne
    lastName: Rodman
    telephone1: 1-617-789-1890
    telephone2: 1-800-445-2588
    password: p4444
    OID: "4"
    email: mrodman@yahoo.com
    adminId: mrodman
    objectName: AdministratorDO
  - 
    subscriber: StarSports
    lastName: Jordan
    telephone1: 1-617-890-7897
    telephone2: 1-800-445-2588
    password: p5555
    OID: "5"
    email: mjordan@yahoo.com
    adminId: mjordan
    objectName: AdministratorDO
  - 
    subscriber: GreatBooks
    lastName: Pippen
    telephone1: 1-781-789-9876
    telephone2: 1-800-445-2588
    password: p6666
    OID: "6"
    email: gpippen@yahoo.com
    adminId: gpippen
    objectName: AdministratorDO
  - 
    subscriber: AxisChemicals
    lastName: Andhrew
    telephone1: 1-781-678-8970
    telephone2: 1-800-445-2588
    password: p7777
    OID: "7"
    email: aandrew@yahoo.com
    adminId: aandrew
    objectName: AdministratorDO
  - 
    subscriber: MediaShop
    lastName: Vincent
    telephone1: 1-786-897-8908
    telephone2: 1-800-445-2588
    password: p8888
    OID: "8"
    email: tvincent@yahoo.com
    adminId: tvincent
    objectName: AdministratorDO
  - 
    subscriber: SmartShop
    lastName: Richard
    telephone1: 1-508-789-6789
    telephone2: 1-800-445-2588
    password: p9999
    OID: "9"
    email: krichard@yahoo.com
    adminId: krichard
    objectName: AdministratorDO
  - 
    subscriber: HomeNeeds
    lastName: Cornell
    telephone1: 1-617-789-8979
    telephone2: 1-800-445-2588
    password: paaaa
    OID: "10"
    email: gconell@yahoo.com
    adminId: gcornell
    objectName: AdministratorDO
  - 
    subscriber: MegaCorp
    lastName: HorstMann
    telephone1: 1-508-791-1267
    telephone2: 1-800-445-2588
    password: p1111
    OID: "11"
    email: shorstmann@yahoo.com
    adminId: shorstmann
    objectName: AdministratorDO
  - 
    subscriber: CornerStore
    lastName: Bob
    telephone1: 1-781-789-9089
    telephone2: 1-800-445-2588
    password: p3333
    OID: "13"
    email: rbob@yahoo.com
    adminId: rbob
    objectName: AdministratorDO
  - 
    subscriber: Cyberdyne
    lastName: Peter
    telephone1: 1-617-789-1890
    telephone2: 1-800-445-2588
    password: p4444
    OID: "14"
    email: speter@yahoo.com
    adminId: speter
    objectName: AdministratorDO
  - 
    subscriber: StarSports
    lastName: Novak
    telephone1: 1-617-890-7897
    telephone2: 1-800-445-2588
    password: p5555
    OID: "15"
    email: pnovak@yahoo.com
    adminId: pnovak
    objectName: AdministratorDO
  - 
    subscriber: GreatBooks
    lastName: Nancy
    telephone1: 1-781-789-9876
    telephone2: 1-800-445-2588
    password: p6666
    OID: "16"
    email: pnancy@yahoo.com
    adminId: pnancy
    objectName: AdministratorDO
  - 
    subscriber: AxisChemicals
    lastName: Michel
    telephone1: 1-781-678-8970
    telephone2: 1-800-445-2588
    password: p7777
    OID: "17"
    email: hmichel@yahoo.com
    adminId: hmichel
    objectName: AdministratorDO
  - 
    subscriber: MediaShop
    lastName: David
    telephone1: 1-786-897-8908
    telephone2: 1-800-445-2588
    password: p8888
    OID: "18"
    email: kdavid@yahoo.com
    adminId: kdavid
    objectName: AdministratorDO
  - 
    subscriber: SmartShop
    lastName: Valnoor
    telephone1: 1-508-789-6789
    telephone2: 1-800-445-2588
    password: p9999
    OID: "19"
    email: pvalnoor@yahoo.com
    adminId: pvalnoor
    objectName: AdministratorDO
  - 
    subscriber: HomeNeeds
    lastName: Smith
    telephone1: 1-617-789-8979
    telephone2: 1-800-445-2588
    password: paaaa
    OID: "20"
    email: wsmith@yahoo.com
    adminId: wsmith
    objectName: AdministratorDO
  - 
    subscriber: MegaCorp
    lastName: Caral
    telephone1: 1-781-789-9876
    telephone2: 1-800-445-2588
    password: p6666
    OID: "21"
    email: gcaral@yahoo.com
    adminId: gcaral
    objectName: AdministratorDO
  - 
    subscriber: CornerStore
    lastName: Hillary
    telephone1: 1-786-897-8908
    telephone2: 1-800-445-2588
    password: p8888
    OID: "23"
    email: phillary@yahoo.com
    adminId: phillary
    objectName: AdministratorDO
  - 
    subscriber: Cyberdyne
    lastName: Philip
    telephone1: 1-508-789-6789
    telephone2: 1-800-445-2588
    password: p9999
    OID: "24"
    email: bphilip@yahoo.com
    adminId: bphilip
    objectName: AdministratorDO
  - 
    subscriber: StarSports
    lastName: Andrea
    telephone1: 1-617-789-8979
    telephone2: 1-800-445-2588
    password: paaaa
    OID: "25"
    email: sandrea@yahoo.com
    adminId: sandrea
    objectName: AdministratorDO
  - 
    subscriber: s4
    lastName: "null"
    telephone1: "null"
    telephone2: "null"
    password: s4
    OID: "26"
    email: "null"
    adminId: s4
    objectName: AdministratorDO
  - 
    subscriber: BigBank
    lastName: administrator
    telephone1: ''
    telephone2: ''
    password: admin
    OID: "82"
    email: ''
    adminId: admin
    objectName: AdministratorDO